components/StoryCards/StoryCard.module.scss

```diff
@@ -1,12 +1,9 @@
 $small-card-image-size: 230px;
+$spacing-20px: $spacing-4 - $spacing-half; // 1.25rem = 20px That's the only hack I can think of which doesn't mess with other spacings
```

Comment:
This looks like a design issue to me 🤔 
It will work for now, but would be nice to figure out together with Hugo or Al to reconsider the spacing system.

cc @iamhugofelix

Reply:
Also, I think it makes sense to rename it to `$spacing-card`, cause we might change the size later and 20px naming will be not relevant anymore.